data-prepper-plugins/key-value-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/keyvalue/KeyValueProcessorConfig.java

```diff
@@ -24,6 +24,7 @@ public class KeyValueProcessorConfig {
     static final String DEFAULT_DELETE_VALUE_REGEX = "";
     static final String DEFAULT_TRANSFORM_KEY = "";
     static final String DEFAULT_WHITESPACE = "lenient";
+    static final boolean DEFAULT_DUPLICATE_VALUES = true;
```

Comment:
In general, we try to keep boolean default values to `false`. Is the current behavior (prior to this commit) to allow duplicate values?

If so, it may be better to invert this. `skip_duplicate_values`. In this way, `false` can be the default.

Reply:
The current behavior is to allow duplicate values. I was originally following the documentation linked in the issue ([here](https://www.elastic.co/guide/en/logstash/current/plugins-filters-kv.html#plugins-filters-kv-allow_duplicate_values)), but the suggestion makes a lot more sense. Will change!